compiler/src/iree/compiler/Codegen/Dialect/CPU/IR/IREECPUTypes.h

```diff
@@ -19,9 +19,21 @@
 
 namespace mlir::iree_compiler::IREE::CPU {
 
-//===----------------------------------------------------------------------===//
-// Utilities.
-//===----------------------------------------------------------------------===//
+/// Representation for all the supported tiling levels. All or just a subset of
+/// them may be available in a valid configuration.
+enum TilingLevel : unsigned {
+  DistributionTiles = 0,
+  CacheParallelTiles = 1,
+  CacheReductionTiles = 2,
```

Comment:
They can be deprecated once I migrate CPU to use the new attribute. For now, I keep it as the same as the TilingConfig.